src/FixedDataTable.js

```diff
@@ -456,6 +456,16 @@ var FixedDataTable = createReactClass({
     );
   },
 
+  _onKeyDown(event) {
+    if(event.key === "PageDown") {
```

Comment:
nit single quotes here & around PageUp.
Also space after 0, in both _onScroll calls